packages/builder/src/components/portal/licensing/AccountLockedModal.svelte

```diff
@@ -23,9 +25,15 @@
     confirmText={"View plans"}
     {onConfirm}
   >
-    <Body size="S"
-      >Due to the free plan user limit being exceeded, your account has been
-      de-activated. Upgrade your plan to re-activate your account.</Body
-    >
+    <Body size="S">
+      {#if lockedBy === LockReason.FREE_TIER}
+        You’re currently on Budibase’s Free plan, which is no longer available
+        for cloud users. Your tenant has been temporarily locked. Please upgrade
+        to a paid plan to keep your data — otherwise it will be removed soon.
+      {:else}
+        Due to the free plan user limit being exceeded, your account has been
```

Comment:
```suggestion
        Due to the Free plan user limit being exceeded, your account has been
```

Just capitalising the "Free" plan to match the paragraph above.